config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
```

Comment:
this file will not be up to GIt, please
cp database.yml database.yml.example
git rm --cached config/database.yml


and in .gitignore add
/config/database.yml